### PR TITLE
fix: [PIE-5085]: changed type as JsonNode

### PR DESCRIPTION
**Pull request by shalini-agr**
Found this issue in QA: 
Error in Redis handler in parsing an execution with both ci and cd stages:

https://console.cloud.google.com/logs/query;query=resource.type%3D"k8s_container"
resource.labels.project_id%3D"qa-setup"
resource.labels.location%3D"us-west1"
resource.labels.cluster_name%3D"qa-private"
resource.labels.namespace_name%3D"harness-nextgen"
labels.k8s-pod%2Fapp%3D"ng-manager"

"PipelineExecutionSummaryCDRedisEventConsumer";cursorTimestamp=2022-08-26T06:25:18.509991115Z?project=qa-setup

The error was happening because of incorrect typecasting of JsonNode.

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/36823)

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/7752dbb3099d2dba3b5d37f5931a5e4c/pull-request/36823)

**Comment by shalini-agr** [comment]
trigger messagecheck

File: 120-ng-manager/src/main/java/io/harness/pms/redisConsumer/PipelineExecutionSummaryCDChangeEventHandler.java [hunk 1]
```diff
@@ -109,7 +108,7 @@ public Record createRecord(String value, String id) {
       }
 
       if (ciExecutionInfo != null) {
```
Comment (naidu-harness) [inline 120-ng-manager/src/main/java/io/harness/pms/redisConsumer/PipelineExecutionSummaryCDChangeEventHandler.java:110]: probably copy paste mistake. if this is cd handler, use cdExecutionInfo

Comment (shalini-agr) [reply 120-ng-manager/src/main/java/io/harness/pms/redisConsumer/PipelineExecutionSummaryCDChangeEventHandler.java:110]: No, this is ciExecutionInfo only, there are also some ci-related fields stored in cd table of timescale like branch, commitId, etc. which are coming from ci stages.

**Comment by sahilhindwani-harness** [comment]
*[`120-ng-manager/src/main/java/io/harness/pms/redisConsumer/PipelineExecutionSummaryCDChangeEventHandler.java` line 110 at r1](https://reviewable.io/reviews/harness/harness-core/36823#-NAO7dpCFNP18wHcpSTf-r1-110:-NAOngaRCMNz6WYO1Nt_:bmqfnf5) ([raw file](https://github.com/harness/harness-core/blob/8fcde793ab7a801f943981432a5873ab4dfeb274/120-ng-manager/src/main/java/io/harness/pms/redisConsumer/PipelineExecutionSummaryCDChangeEventHandler.java#L110)):*
<details><summary><i>Previously, shalini-agr (Shalini Agrawal) wrote…</i></summary><blockquote>

No, this is ciExecutionInfo only, there are also some ci-related fields stored in cd table of timescale like branch, commitId, etc. which are coming from ci stages. 
</blockquote></details>

I think this is wrong. We should not be storing CI information for CD. This is introducing tight coupling between cd and ci. @abhinavmittal-harness , do you know why we are doing this?

**Review by sahilhindwani-harness — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/36823)* status: 1 file(s) to review. 1. unresolved discussions. 1 of 2 approvals obtained.

**Review by naidu-harness — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/36823)* status: :shipit: complete! ✓

**Pull request merged**